src/apps/seo/src/views/RedirectsManager/RedirectActions/RedirectFilter/RedirectFilter.js

```diff
@@ -1,41 +1,26 @@
-import React, { Component } from "react";
+import React, { useState, useRef } from "react";
```

Comment:
useRef looks like it's not being used. should this be removed?

Reply:
removed